### DISCUSSION TITLE
Run evaluation of nnUNetv2 on test data

### DISCUSSION THREAD

**Discussion by rajanish4** [MIC-DKFZ/nnUNet#1378] [Q&A] [answered]
Hi,

I have been able to make predictions on test data, however, for evaluation i couldnt find documentation. Can someone help me with the command or function used for evaluation on test data?

**Comment by FabianIsensee** [comment, marked as answer]
`nnUNetv2_evaluate_folder` or `nnUNetv2_evaluate_simple`. Use -h for usage instructions
